src/Ocelot.Provider.Polly/PollyQoSProvider.cs

```diff
@@ -1,54 +1,81 @@
-using Ocelot.Configuration;
-using Ocelot.Logging;
-using Ocelot.Provider.Polly.Interfaces;
-using Polly.CircuitBreaker;
-using Polly.Timeout;
-
-namespace Ocelot.Provider.Polly
-{
-    public class PollyQoSProvider : IPollyQoSProvider
-    {
-        public PollyQoSProvider(DownstreamRoute route, IOcelotLoggerFactory loggerFactory)
-        {
-            AsyncCircuitBreakerPolicy circuitBreakerPolicy = null;
-            if (route.QosOptions.ExceptionsAllowedBeforeBreaking > 0)
+using System.Net;
```

Comment:
Please, show me the current Usings settings in Visual Studio which put system namespaces on top of all.
We need to use simple Remove and Sort refactoring...

Reply:
Sorry, I keep forgetting your settings

Reply:
Not mine only 🤣  for whole community!
I don't see any benefits at all to shuffle usings lines. Keeping this simplest is better. Remove and Sort refactoring is ideal candidate to keep usings good.